packages/dev/core/src/AudioV2/webAudio/components/spatialWebAudioUpdaterComponent.ts

```diff
@@ -0,0 +1,48 @@
+import { PrecisionDate } from "../../../Misc/precisionDate";
+
+/** @internal */
+export class _SpatialWebAudioUpdaterComponent {
+    private _autoUpdate = false;
+    private _lastUpdateTime: number = 0;
+
+    /** @internal */
+    public minUpdateTime = 0;
+
+    /** @internal */
+    public constructor(parent: { update: () => void }, autoUpdate: boolean, minUpdateTime: number) {
```

Comment:
shouldn't autoUpdate default to `true`?

Reply:
It defaults to `true` in the top-level classes. See https://github.com/BabylonJS/Babylon.js/blob/2ab407b5008ba66c45f6750f2f91255eb0ce3918/packages/dev/core/src/AudioV2/webAudio/webAudioStaticSound.ts#L24 for example.

In this utility class it's a little more efficient to initialize it to `false` and early out of the constructor if the default value of `true` is explicitly set to `false`.

Reply:
being an internal class, initialized by other classes, I think it's simply more readable, but I am fine with that as long as it stays internal.